modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/index/name/RankingIndexNameBuilderImplTest.java

```diff
@@ -35,24 +35,25 @@ public class RankingIndexNameBuilderImplTest {
 
 	@Test
 	public void testMultiTenancy() {
-		_assertIndexName(
-			2021, companyId -> "liferay-" + companyId,
-			"liferay-2021-search-tuning-rankings");
-	}
-
-	private void _assertIndexName(
-		long companyId, IndexNameBuilder indexNameBuilder, String expected) {
-
 		RankingIndexNameBuilderImpl rankingIndexNameBuilderImpl =
 			new RankingIndexNameBuilderImpl();
 
 		ReflectionTestUtil.setFieldValue(
-			rankingIndexNameBuilderImpl, "_indexNameBuilder", indexNameBuilder);
+			rankingIndexNameBuilderImpl, "_indexNameBuilder",
+			new IndexNameBuilder() {
```

Comment:
@W2jx Another thing, the original code creates the `IndexNameBuilder` using `companyId -> "liferay-" + companyId`, which is a lambda, why change it back to anonymous class?

Reply:
Because input parameters of method setFieldValue are (Object, string, Object), and lambda cannot be assigned to Object.